### ISSUE TITLE
Error: If both baselineFilepath and outDir are configured new baseline file is written to a wrong directory

### ISSUE THREAD

**Issue by viktorijasolovjova** [relative-ci/bundle-stats#3975]
**Problem**
With the following configuration (providing both `baselineFilepath` and `outDir` paths and also selecting to save the new baseline file):

```javascript
  config.plugins.push(
    new BundleStatsWebpackPlugin({
      baselineFilepath: '../../../webpack-bundle-stats/baseline.json', // relative to the webpack output path
      compare: true,
      baseline: true,
      outDir: '../..'  // relative to the webpack output path, results in Cadenza_Web_Frontend/target/
  })
  );
``` 

the baseline is read from the right path, but is written to a wrong one afterwards.

For us the paths resolves like this:
- webpack output path: `<project-path>/target/webapp/bundles`
- outDir: `<project-path>/target/`
- baselineFilepath when reading: `<project-path>/webpack-bundle-stats/baseline.json`
- baselineFilepath when writing it: `<project-path>/target/webapp/webpack-bundle-stats/baseline.json`

**Possible cause**
It looks like the asset added [here](https://github.com/relative-ci/bundle-stats/blob/3535ce788c661c8c5ab0b167c9a0faea56a5da89/packages/cli-utils/src/reports.ts#L148): 
- has the right absolute path `baselineAbsolutePath = '<project-path>/webpack-bundle-stats/baseline.json'`, 
- but still packs the asset with the key `baselinePath = '../webpack-bundle-stats/baseline.json'` to the `newAssets`, 
- so Webpack afterwards decides to resolve it relatively to its output path `<project-path>/target/webapp/bundles`,
- which resolves to `<project-path>/target/webapp/webpack-bundle-stats/baseline.json`.

**Comment by vio** [comment]
@viktorijasolovjova thank you for creating the issue! Webpack requires a relative path for the asset, though it looks like we are not computing the write file path correctly. 

We are planning to write all the files outside of the webpack compilation (not really helpful to include them in the stats), but will have a look at his issue soon.

**Comment by vio** [comment]
A fix was released in [`v4.9.2`] (https://github.com/relative-ci/bundle-stats/releases/tag/v4.9.2).

Thank you for the detailed report! 🙇

**Comment by viktorijasolovjova** [comment]
Thank you for fixing, it works for us as expected now! (y)
